### DISCUSSION TITLE
[Feature Request] Add support for yaml as datasource

### DISCUSSION THREAD

**Discussion by 4n4nd** [grafana/grafana-infinity-datasource#271] [Ideas]
Similar to pulling data from jsons, I think being able to pull data from yamls should be a great addition

**Comment by yesoreyeram** [comment]
I don't think any APIs that expose data via YAML. Do you any example APIs?

**Comment by 4n4nd** [comment]
Not APIs per se, but I was thinking it would be neat to be able to link a yaml manifest ([example](https://github.com/operate-first/apps/blob/ce6032596d86560c1c19e5cc9a3dcb9b3ef814da/cluster-scope/base/core/namespaces/opf-alertreceiver/namespace.yaml#L6)) and be able to pull data from it directly. Since a lot of Kubernetes/Docker/Container manifests use the yaml format, it would be awesome to be able to pull data from such a manifest and display it in a dashboard along with other metrics.

This would just be text data ofc and no timeseries.
